assets/js/app/Data.js

```diff
@@ -143,13 +217,24 @@ LocusZoom.Data.LDSource = function(url) {
         };
     };
 };
+LocusZoom.Data.LDSource.SOURCE_NAME = "LDLZ";
 
-LocusZoom.Data.GeneSource = function(url) {
-    this.url = url;
+LocusZoom.Data.GeneSource = function(init) {
+    if (LocusZoom.isString(init)) {
+        this.url = init;
+        this.params = {};
+    } else {
+        this.url = init.url;
+        this.params = init.params || {};
+    }
+    if (!this.url) {
+        throw("GeneSource not initialized with required URL");
+    }
 
+    var that = this;
```

Comment:
[See other note regarding this/that/bind](https://github.com/statgen/locuszoom/pull/7/files#r48358203)